src/core/thread/mle.cpp

```diff
@@ -2139,10 +2139,6 @@ ThreadError Mle::HandleChildIdResponse(const Message &aMessage, const Ip6::Messa
             mNetif.GetPendingDataset().Set(pendingTimestamp, aMessage, offset + sizeof(tlv), tlv.GetLength());
         }
     }
-    else
```

Comment:
Yes, I understand.  However, is there any practical value to that?  Whatever Active Dataset the network has (even with a timestamp of 0), the device should synchronize with that to ensure that it has all the same active parameters.